djangocms_content_expiry/static/djangocms_content_expiry/css/actions.css

```diff
@@ -38,4 +38,17 @@ span.svg-juxtaposed-font {
     line-height: 1rem;
     vertical-align: 20%;
     margin-top: -2px !important;
+}
+
+/*
+Compliance number is not a mandatory field and is certain scenarios read-only
+so below is required to make the labels and value field look consistent with other fields
+*/
+.field-compliance_number label {
```

Comment:
This would probably be better suited to its own file to split the admin actions from any form styles.